### ISSUE TITLE
Pretty cool project, only missing a slideshow feature!

### ISSUE THREAD

**Issue by sardaukar** [viktorstrate/photoview#51]
Thanks for this project, it's really cool to see a simple folder based approach to photo browsing! The sharing photos are the icing on the cake.

I currently use ThumbsUp to generate static gallery websites, and it has a really cool slideshow feature that you can see for yourself at https://thumbsup.github.io/demos/themes/mosaic/albums/2015-March.html (just press the play button on the top right). 

It'd be awesome if Photoview had something similar!

**Comment by viktorstrate** [comment]
I think a slideshow feature would be great. 👍

The demo link, looks like a good source of inspiration.

**Comment by imler** [comment]
Another feature I would love to see.

Just my two cents - there should be an option in settings to configure the time of transition to the next photo.

A slider or something else directly in the viewer to easily change that time for the current viewing session would be handy, but at least a global setting is a must for this feature.

**Comment by dstrelnikov** [comment]
Yes, slideshow feature would be really amazing! I like how it is implemented in Gwenview that one can control the slideshow via standard play/pause, next and prev buttuns (via MPRIS).

**Comment by Manfred73** [comment]
This is one of the view applications I have found that allow browsing photos by folder structure. That's really cool. Also love the sharing options and possibility to add multiple users.
As slideshow option for sharing would be awesome.

**Comment by tmanok** [comment]
Please enable a slideshow feature! Migrating from Piwigo due to video support, timeline feature, and file upload issues (mostly due to the upload feature not liking certain formats and sizes). One of the pain points for me is the lack of a slideshow feature to plug in a laptop to a TV and show the family or groups of friends. 

**If a slideshow is implemented, it would be important to only count the seconds after the full resolution photo has been displayed, not just the compressed image.** Pre-caching in the browser might also be a good idea after the slideshow has commenced to improve smoothness. 

Thank you very very very much!

Tmanok

**Comment by kkovaletp** [comment]
Unfortunately, we haven't had a UI maintainer for a while, so many UI-related PRs with features and fixes are waiting for code review.
If you know a Senior UI developer with good-enough experience in React + GraphQL technology stack, and willing to help us with code reviews, UI refactoring/bug-fixes, and new feature development, as a UI maintainer, the Photoview community would be happy to finally get those things managed
